index.bs

```diff
@@ -24,9 +46,66 @@ Please join us in the <a href="https://github.com/terinjokes/console-spec/issues
 <h2 id="logger-function">Logger Function</h2>
 The logging function accepts datum and the logLevel, which is stored or displayed in a platform appropriate way.
 
-TODO: Describe the interaction of format specifiers and object inspection with respect to what is stored or displayed
-
-TODO: Add notes to give examples of how a platform might implement the side effects of this logging function.
+Each call of the Logger function prints a line which is ended by a newline.
+If the Logger function is called with no arguments, no output is written to the console.
+If the printing function gets multiple elements, it prints them separated by a space. It is possible to pass format specifiers to the function as the element of the arguments that should get printed.
+
+The Logger(logLevel, multipleArgs) function must run these steps:
+
+<emu-alg>
+  1. Let _firstElement_ be the first element of _multipleArgs_.
+  1. Let _rest_ be all elements following _firstElement_ in _multipleArgs_.
+  1. Let _formatter_ be a recursive function.
+  1. If no _firstElement_ given, do not print any output. Abort these steps.
+  1. If just _firstElement_ given, print _firstElement_ to the console. Abort these steps.
+  1. If _firstElement_ does not contain any format specifier, print _multipleArgs_ separated by one space to the console. Abort these steps.
+  1. Call _formatter_ with _multipleArgs_:
+    1. Let _target_ be the first element of _multipleArgs_.
+    1. Let _current_ be the second element of _multipleArgs_.
+    1. Find the first possible format specifier from the left to the right in _target_.
+      1. If it is `%s`, let _converted_ be the result of new String(_current_).
```

Comment:
do `ToString(_current_)` instead. `new String(_current_)` implies that by modifying `global.String` I can modify this result.